src/components/ContentWidgets/ToDoList/ToDoListEditor.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+import { ToDoListItem } from './ToDoList';
+
+export const ToDoListEditor: React.FC<WidgetEditorProps> = ({
+    originalContent,
+    editedContent,
+    setEditedContentOnChange
+}) => {
+    let listContent = null;
+
+    // let findItemIndex = (findItemID: number, copyEditedContent: ToDoListItem[]) => {
+        
+    //     for (let i = 0; i < copyEditedContent.length; i++) {
+    //         if (findItemID == copyEditedContent[i].toDoListItem_order) {
+    //             return i;
+    //         }
+    //     }
+    //     return -1;
+    // }
+
+    if (editedContent.toDoList_content != null) {
+        listContent = editedContent.toDoList_content.map((item) => {
+            
+            let copyEditedContent = [...editedContent.toDoList_content!!];
```

Comment:
What does the double bang do?  Is this even valid javascript? When I try it in my console I get an unexpected token issue.

Reply:
I haven't tried to run the code but I would think it would crash.